cdktf init logs an error

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### cdktf & Language Versions

- 0.6
<!--- Please make note of the version you are using for the `hashicorp/terraform-cdk` and language bindings you are using.
If you are not running the latest version, please upgrade because your issue may have already been fixed. --->

### Affected Resource(s)

cli

<!--- Please list the affected resources. --->

### Debug Output

```
  You can also build any module or provider locally. Learn more https://cdk.tf/modules-and-providers

========================================================================================================

[2021-09-16T10:01:40.896] [ERROR] default - Could not send telemetry data: TypeError: explanatoryComment.replaceAll is not a function
```

<!---
Please provide a link to a GitHub Gist containing the complete debug output. Please do NOT paste the debug output in the issue; just paste a link to the Gist.
--->

### Expected Behavior

no error log output

<!--- What should have happened? --->

### Actual Behavior

<!--- What actually happened? --->

### Steps to Reproduce

`cdktf init --template typescript --local`

<!--- Please list the steps required to reproduce the issue. --->

### Important Factoids

<!--- Are there anything atypical about your accounts that we should know? --->

### References

<!---
Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor documentation? For example:
--->

- #0000

--- a/packages/cdktf-cli/lib/checkpoint.ts
+++ b/packages/cdktf-cli/lib/checkpoint.ts
@@ -91,7 +91,7 @@ function getId(
   const _uuid = uuidv4();
   const _idFile = {} as Record<string, string>;
   if (explanatoryComment) {
-    _idFile["//"] = explanatoryComment.replaceAll("\n", " ");
+    _idFile["//"] = explanatoryComment.replace(/\n/g, " ");
   }
   _idFile[key] = _uuid;
 

